examples/Main.hs

```diff
@@ -46,6 +52,53 @@ example2 = plam $ \x -> pmatch x $ \case
   PLeft n -> n + 1
   PRight n -> n - 1
 
+data SampleRecord f = SampleRecord {
```

Comment:
Why not replace `f` with `s` and have all the fields be `Term s ...` to make it match other Plutarch types?